shared/constants/settings.js

```diff
@@ -1,11 +1,63 @@
 // @flow
 
 export type PlanLevel = 'Basic' | 'Gold' | 'Friend'
-export const plans: Array<PlanLevel> = ['Basic', 'Gold', 'Friend']
+const plans: Array<PlanLevel> = ['Basic', 'Gold', 'Friend']
 
 export type PaymentInfo = {
   name: string,
   last4Digits: string,
   isBroken: boolean,
 }
 
+const levelToPrice: {[key: PlanLevel]: string} = {
+  'Basic': 'Free',
+  'Gold': '$7/mo',
+  'Friend': '$9/mo',
+}
+
+const levelToPriceLong: {[key: PlanLevel]: string} = {
+  'Basic': 'Free',
+  'Gold': '$7/month',
+  'Friend': '$9/month',
+}
+
+const levelToStars: {[key: PlanLevel]: number} = {
+  'Basic': 1,
+  'Gold': 3,
+  'Friend': 5,
+}
+
+const levelToSpace: {[key: PlanLevel]: string} = {
+  'Basic': '10GB',
+  'Gold': '50GB',
+  'Friend': '250GB',
+}
+
+function levelToDetails (p: PlanLevel) {
+  return `You will be able to use up to ${levelToSpace[p]} of data`
```

Comment:
I think you should add a period to the end of the sentence here to match the mockups.